script.js

```diff
@@ -0,0 +1,366 @@
+    };
+
+    recipes.push(newRecipe);
+    enrichRecipesWithPrices();
+    renderRecipes(recipes);
+    newForm.innerHTML = '';
+    addNewButton.style.display = 'block';
+  });
+
+  newForm.appendChild(addRecipe);
+}
+
+function startPageTimer() {
+  const timerElement = document.querySelector('#page-timer');
+  if (!timerElement) {
+    console.error("Element with id '#page-timer' not found.");
+    return;
+  }
+
+  let seconds = 0;
+
+  function updateDisplay() {
+    const hours = Math.floor(seconds / 3600).toString().padStart(2, '0');
+    const minutes = Math.floor((seconds % 3600) / 60).toString().padStart(2, '0');
+    const secs = (seconds % 60).toString().padStart(2, '0');
+    timerElement.textContent = `${hours}:${minutes}:${secs}`;
+  }
+
+  setInterval(() => {
+    seconds++;
+    updateDisplay();
+  }, 1000);
+
+  updateDisplay();
+}
+
+startPageTimer();
+
+document.querySelector('#start-timer').addEventListener('click', startCustomTimer);
+
+let isTimerRunning = false;
+let timerInterval = null;
+
+function startCustomTimer() {
+  const timerInput = document.querySelector('#timer-input').value.trim();
+  const timerDisplay = document.querySelector('#timer-display');
+  const timeSound = document.querySelector('#alarm-sound');
+  const finishSound = document.querySelector('#finish-sound');
+  const startButton = document.querySelector('#start-timer');
+
+  const minutes = parseInt(timerInput, 10);
+
+  if (isNaN(minutes) || minutes <= 0) {
+    alert('Please enter a valid time in minutes (greater than 0)');
+    return;
+  }
+
+  let totalSeconds = minutes * 60;
+
+  function updateDisplay() {
+    const hours = Math.floor(totalSeconds / 3600).toString().padStart(2, '0');
+    const mins = Math.floor((totalSeconds % 3600) / 60).toString().padStart(2, '0');
+    const secs = (totalSeconds % 60).toString().padStart(2, '0');
+    timerDisplay.textContent = `${hours}:${mins}:${secs}`;
+  }
+
+  function stopTimer() {
+    clearInterval(timerInterval);
+    timerDisplay.textContent = '00:00:00';
+    finishSound.play();
+    startButton.textContent = 'Start timer';
+    isTimerRunning = false;
+  }
+
+  if (isTimerRunning) {
+    stopTimer();
+    return;
+  }
+
+  timerInterval = setInterval(() => {
+    totalSeconds--;
+    if (totalSeconds < 0) {
+      stopTimer();
+    } else {
+      isTimerRunning = true;
+      timeSound.play();
+      updateDisplay();
+    }
+  }, 1000);
+
+  isTimerRunning = true;
+  startButton.textContent = 'Stop Timer';
+  updateDisplay();
+  timerInput.textContent = 0;
+}
+
+const addNewButton = document.querySelector('#add-new-recipe');
+const sortButton = document.querySelector('#sort-button');
+const searchButton = document.querySelector('#search-button');
+
+if (addNewButton) addNewButton.addEventListener('click', createRecipeForm);
+if (sortButton) sortButton.addEventListener('click', sortByAmountIngredients);
+if (searchButton) searchButton.addEventListener('click', findRecipeByWord);
```

Comment:
If you were using an HTML `<form>`, you could listen to the `sumbit` event on that form. This would allow users to use the keyboard (and Enter key) to submit your form and perform the search.